lib/indico.js

```diff
@@ -30,9 +31,15 @@ indico.Collection = function(collectionName, config) {
 }
 
 indico.collections = function(config) {
-  return make_request('/custom/collections', {}, false, config);
+  return makeRequest('/custom/collections', {}, false, config);
 }
 
+indico.relevance = function(data, queries, config) {
```

Comment:
Because the queries argument isn't optional -- it's required. Setting a rigid format like we have in the settings file means that we make it hard for ourselves to support APIs with new input types (like this one).